config/vufind/config.ini

```diff
@@ -165,6 +165,17 @@ allowSavedSearches = true
 nonJavascriptSupportEnabled = false
 ; Generator value to display in an HTML header <meta> tag:
 generator = "VuFind 8.0.1"
+; headTitle version
+; pre = set the information as prefix before header
+; pos = set the information as postfix after header
```

Comment:
I would prefer "post" to "pos" here -- I think "pos" could be misunderstood as "position" but "post" is more clear.